corona-warn-app/cwa-app-android
# [Feature] Support revocation of single certificates (EXPOSUREAPP-12455)

This PR addresses [EXPOSUREAPP-12455](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12455).

Revocation of single certificates story.

How to test:
- Scan in a valid VC (MockCloud Environment)
- use CLI tool to revoke a certficate (newest version needed):
 _prompts for entering the barcode data, similar to `cwa dcc verify`
 then prompts for revoking or granting the DCC repeatedly_:
_cwa dcc rl -e mock-cloud --data -i_

- update the revocation list inside the dev menu or wait 24 h
- check if the certificate is revoked
- check if texts and UI matches to figma
- check if acceptance criteria are fulfilled
